File: packages/flutter_shared/lib/tools/pomodoro_tool.dart
/// 番茄钟工具
class PomodoroTool {
  PomodoroTool._();

  static const defaultWork = 25;
  static const defaultBreak = 5;
  static const defaultLongBreak = 15;
  static const longBreakInterval = 4;

  /// 阶段描述
  static String phaseLabel(int phaseIndex) {
    return switch (phaseIndex) {
      0 => '工作中',
      1 => '短休息',
      2 => '长休息',
      _ => '',
    };
  }
}

File: miniapp/src/utils/tools/pomodoroTool.ts
/**
 * 番茄钟工具
 * 移植自 flutter_shared/tools/pomodoro_tool.dart
 */

export const PomodoroTool = {
  defaultWork: 25,
  defaultBreak: 5,
  defaultLongBreak: 15,
  longBreakInterval: 4,

  phaseLabel(phaseIndex: number): string {
    switch (phaseIndex) {
      case 0: return '工作中';
      case 1: return '短休息';
      case 2: return '长休息';
      default: return '';
    }
  },
};
